docs/topics/rn-known-issues.adoc

```diff
@@ -18,21 +18,17 @@ For a complete list of all known issues, see the list of link:https://issues.red
 
 |link:https://issues.redhat.com/browse/WINDUP-3138[WINDUP-3138]
 |CodeReady Workspace plugin
-|The plugin does not work with CodeReady Workspace 2.10.
+|The plugin works with CodeReady Workspace 2.09, but not version 2.10.
 
-|link:https://issues.redhat.com/browse/WINDUP-3019[WINDUP-3019]
-|CLI
-|Bash completion feature displays an `mta-cli not found` error.
+|link:https://issues.redhat.com/browse/WINDUP-3165[WINDUP-3165]
+|Web console
+|MTA analysis fails on OpenShift Container Platform when custom labels are uploaded and advanced options are enabled.
```

Comment:
This is strange. Does this happen on all versions of OpenShift or just the latest version? 

The second part is a dangler. It sounds like you are referring to OpenShift labels and advanced options. I had to check the bug to see what you meant. Rewrite the clause so that it is not as ambiguous.

It is odd that this bug was not detected before release. Labels and advanced options are basic functionality for MTA and they should have been tested on all supported environments. Sounds like regression tests are not being run.